### PR TITLE
Update for newer form of base driver timeouts

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.7",
-    "appium-base-driver": "^2.0.3",
+    "appium-base-driver": "^2.0.9",
```
Comment (jlipps) [inline package.json]: was this a breaking change in base-driver?

**Comment by jlipps** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (imurchie) [inline package.json]: It's not a breaking change. It's just something additional that can be made use of here.

Comment (jlipps) [inline package.json]: ok 👍

**Pull request merged**